cykelplanen/build.gradle

```diff
@@ -21,7 +21,7 @@ android {
         applicationId "com.spoiledmilk.cykelsuperstier"
         minSdkVersion 16
         targetSdkVersion 27
-        manifestPlaceholders = [HOCKEYAPP_APP_ID: ""]
+        manifestPlaceholders = [HOCKEYAPP_APP_ID: "cec6c0e595db43ffa1e9a60bfab16161"]
```

Comment:
is this ok to have in the repo?

Reply:
This is okay, to have in the repository. 
Is just a way for HockeyApp to recognize the app when uploaded. Only superusers in HockeyApp can upload.